File: CloudBackup.Model/Configuration.cs

namespace CloudBackup.Model
{
    public class Configuration : EntityBase
    {
        public string InstanceId { get; set; } = null!;

        public string UserName { get; set; } = null!;

        public string Email { get; set; } = null!;

        public string Password { get; set; } = null!;

        public long UtcOffsetTicks { get; set; }

        public ConfigurationStatus ConfigurationStatus { get; set; }

        public string? ErrorMessage { get; set; }
    }

    public enum ConfigurationStatus
    {
        Created = 0,
        Processed = 1,
        Failed = 2
    }
}

File: CloudBackup.Management/ClientApp/src/app/classes/Configuration.ts
import {TimeSpan} from "./TimeSpan";

export class Configuration {
    public ConfigurationStatus: string;
    public Email: string;
    public InstanceId: string;
    public Password: string;
    public UtcOffset: TimeSpan;
    public UserName: string;

    constructor() {
        this.ConfigurationStatus =
        this.Email =
        this.InstanceId =
        this.Password =
        this.UserName = "";
    }
}
